t/unit/app/test_builtins.py

```diff
@@ -104,6 +107,11 @@ def test_apply_async_eager(self):
         self.task.apply_async((1, 2, 3, 4, 5))
         self.task.apply.assert_called()
 
+    def test_apply_async_eager_serialization_failure(self):
+        self.task.apply = Mock(name='apply')
+        with pytest.raises(EncodeError):
+            self.task.apply_async((1, 2, 3, 4, {1}))
+
```

Comment:
Is there somewhere better this can go than in the groups test case?

Reply:
Probably, I wasn't sure where exactly it should live. Any suggestions?

Reply:
The test case should live under test_tasks.py in my opinion.